homeassistant/components/eq3btsmart/manifest.json

```diff
@@ -0,0 +1,27 @@
+{
+  "domain": "eq3btsmart",
+  "name": "eQ-3 Bluetooth Smart Thermostats",
+  "bluetooth": [
+    {
+      "local_name": "CC-RT-BLE",
+      "connectable": true
+    },
+    {
+      "local_name": "CC-RT-M-BLE",
+      "connectable": true
+    },
+    {
+      "local_name": "CC-RT-BLE-EQ",
+      "connectable": true
+    }
+  ],
+  "codeowners": ["@eulemitkeule", "@dbuezas"],
```

Comment:
It would great to get a 👍  from dbuezas here before merging this

Reply:
Sure, I wrote him a DM and asked if he wanted to be listed before, told him to react here now.

Reply:
I'm ok with being a co-code owner👍
Awesome work @EuleMitKeule